data-prepper-plugins/opensearch/src/main/resources/index-template/logs-otel-v1-index-standard-template.json

```diff
@@ -78,50 +149,56 @@
           "path": "observedTimestamp"
         },
         "traceId": {
-          "ignore_above": 256,
-          "type": "keyword"
+          "type": "keyword",
+          "ignore_above": 32
         },
         "spanId": {
-          "ignore_above": 256,
-          "type": "keyword"
+          "type": "keyword",
+          "ignore_above": 16
```

Comment:
Why are we limiting to 16 chars here and 32 in traceId?

Reply:
First, this is not a limitation of storage, just of the indexing.  [W3C Trace Context](https://www.w3.org/TR/trace-context/) specifies the trace id to be an array of 16 bytes and the span id to be an array of 8 bytes. With hex-encoding this produces 32 and 16 characters respectively. Using [B3 context](https://github.com/openzipkin/b3-propagation) would provide the same length. So it seems to be a reasonable choice.